python/Platform/Mac/PythonEntitlements.plist

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
these ones are about the O3DE_SDK right? can this file be moved together with that tool

Reply:
This is for the Python framework executables.

Reply:
gotcha, then ok to leave it here